src/web/components/MostViewed/MostViewedFooter/MostViewedFooterData.tsx

```diff
@@ -48,8 +54,9 @@ export const MostViewedFooterData = ({
 	sectionName,
 	pillar,
 	ajaxUrl,
+	design,
 }: Props) => {
-	const url = buildSectionUrl(ajaxUrl, sectionName);
+	const url = buildSectionUrl(ajaxUrl, pillar, sectionName);
```

Comment:
Feel free to take or leave this but I think now that we're passing 3 items in it might make sense to refactor this to pass an object? Eg

```
buildSectionUrl({ajaxUrl, pillar, sectionName});
```

and then 

```
function buildSectionUrl({
    ajaxUrl,
    pillar,
    secionName,
}: {
	ajaxUrl: string;
	pillar: CAPIPillar;
    sectionName?: string;
}) ...
```